src/events/EventsApp.js

```diff
@@ -90,14 +91,46 @@ const EventsApp = observer(class App extends React.Component {
         this.view.ui.add(searchExpand, "top-right");
         this.view.ui.add(legend, "bottom-right");
         this.view.ui.move("zoom", "top-right");
+        this._initAutoRefreshInterval();
+        App._fixLayerOrder(this.view);
       });
   }
 
+  static _fixLayerOrder(mapView) {
+    const layers = mapView.map.layers;
+    const iSpeed = layers.items.findIndex(lyr => lyr.id === 'speed');
+    if (iSpeed > 0) {
+      const speedLayer = layers.getItemAt(iSpeed);
+      layers.reorder(speedLayer, 0);
+    }
+  }
```

Comment:
I haven't found a better way to enforce layer order from the start. Updating layer settings in the config file didn't help.

Reply:
how does it look in the ui?
I'm good with it if there is no impact

Reply:
![image](https://user-images.githubusercontent.com/25718561/74035774-fa045c80-49cb-11ea-91ff-513600625fae.png)

Icons are rendered on top of the speed lines